src/style.css

```
@@ -1 +1,69 @@
 /* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  padding: 0;
+}
```

```suggestion
body {
  margin: 0;
}
```
move font-family to html selector
body has default margin